model/office/file_by_key.go

```diff
@@ -0,0 +1,44 @@
+package office
+
+import (
+	"bytes"
+
+	"github.com/cozy/cozy-stack/model/instance"
+	"github.com/cozy/cozy-stack/model/vfs"
+)
+
+func GetFileByKey(inst *instance.Instance, key string) (*vfs.FileDoc, error) {
+	detector, err := GetStore().GetDoc(inst, key)
```

Comment:
What's the idea behind the `detector` name? I would expect a `doc` variable here (as is used in other parts of the office code).

Reply:
It's an id+rev+md5sum used to detect if a conflict has happened on this file. `doc` is used when we have a full document with the other fields (like name and size).